code/src/main/java/com/onlinecontacttracing/servlets/get-data/GetPositiveUserInfoServlet.java

```diff
@@ -0,0 +1,85 @@
+package com.onlinecontacttracing.servlets;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import com.onlinecontacttracing.storage.PotentialContact;
+import static com.googlecode.objectify.ObjectifyService.ofy;
+import com.onlinecontacttracing.storage.PositiveUser;
+import java.util.Optional;
+import java.util.ArrayList;
+
+@WebServlet("/get-positve-user-info")
+public class GetPositiveUserInfoServlet extends HttpServlet {
+
+  /*
+  * This servlet will retrieve data from the People Api
+  * Additionaly it will forward the request to the servlet for Calendar Api
+  * Once both are done, the servlet will merge contact data sets
+  */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    updateUser(request.getParameter("idToken"));
+
+    // Execute runnable to get people data
+    ArrayList<PotentialContact> contactsFromPeople = new ArrayList<PotentialContact>();
+    Thread peopleInfo = new Thread(new GetPeopleData(contactsFromPeople));
+    peopleInfo.start();
+
+    // Forward to servlet to retrieve calendar data
+    try {
+      request.getRequestDispatcher("/get-positive-user-calendar-info").forward(request,response);
```

Comment:
Similar to what you did with getPeopleData, rather than doing a servlet forward here, let's create a runnable that we can call from both servlets.